src/exporthtml.ts

```diff
@@ -36,6 +36,10 @@ function generateTranscript<T extends ReturnTypes>(messages: discord.Message[],
 
     document.getElementById('guildname')!.textContent = channel.guild.name;
     document.getElementById('ticketname')!.textContent = channel.name;
+    document.getElementById('tickettopic')!.textContent = `This is the start of the #${channel.name} channel.`;
+    if (channel.topic && channel.topic != null) {
+        document.getElementById('tickettopic')!.innerHTML = channel.topic;
```

Comment:
if you are going to use innerHTML, make sure to pass channel.topic into formatContent, or set textContent instead.
This way no xss can occur.